ios/NativeSigner/Modals/ExportKeys/ExportMultipleKeysModal+ViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ExportMultipleKeysModal+ViewModel.swift
+//  NativeSigner
+//
+//  Created by Krzysztof Rodak on 24/10/2022.
```

Comment:
Do we keep those for all files? Git remembering who created what and for legal reasons only company copyright may be required...

Reply:
I’ll issue PR with change for all files and reconfigure Xcode to add new CP info as agreed thanks for reminding!